docs/migration/2_0.md

```diff
@@ -358,3 +359,6 @@ This component was created with easy usage in mind. The only thing you have to d
 | `RegisterFormComponent` | method | - | `registerUser` |
 | `CustomFormValidators` | validator | `emailDomainValidator` | removed |
 | `CustomFormValidators` | validator | `matchPassword` | removed |
+
+## Store Finder Changes
+Paramater `radius` of `googleMaps?` parameter in `StoreFinderConfig` is now required.
```

Comment:
I would change that. To only say that the radius is now configurable through the config. It is no longer required after the latest config changes ;)